package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "private": true,
+  "scripts": {
```

Comment:
- [ ] Currently, there's no way to start this project. Please make sure it has scripts like `start`. 😸 
_You can add the following:_
```
"start": "webpack --config webpack.config.js --watch"
```

Reply:
@jasonfelice Do you mean I can add the script given above? Do I need to install any other dependencies?

Reply:
- [ ] The webpack dev server still doesn't work, please follow the steps in [this section](https://webpack.js.org/guides/development/#using-webpack-dev-server) of the webpack documentation about using the webpack dev server.

[Hint] The suggestions that code reviewers give are not always right you suppose to test it first and if it doesn't work search for the solution yourself. For example, the correct script you suppose to add as mentioned in the link I gave you above is this:
```json
"start": "webpack serve --open",
```

[Hint]: While applying the dev server to your project I faced another error with the output bundle file so Kindly ensure that you change the output section in your `webpack.config.js` file to this:
```javascript
output: {
    filename: '[name].bundle.js',
    path: path.resolve(__dirname, 'dist'),
    clean: true,
 },
```

Reply:
Thank you so much @Omar-Muhamad for clarifying my confusion regarding **Build using Webpack dev Server**.
I appreciate your time and effort in bringing me the solution.

Reply:
My Pleasure, 
Have a great day